build.gradle

```diff
@@ -1,6 +1,8 @@
 plugins {
-    id 'com.gradle.build-scan' version '1.16'
-    id 'nu.studer.plugindev' version '1.0.12'
```

Comment:
The latest version of `nu.studer.plugindev` is causing a conflict with one of the kotlin dependencies it seems, it's trying to create a task called `pluginUnderTestMetadata` but it already exists. The easiest solution would be to release a version of `nu.studer.plugindev` that is more tolerant of task names conflicts